dev/com.ibm.ws.microprofile.health.2.0_fat/fat/src/com/ibm/ws/microprofile/health20/fat/MultipleHealthCheckTest.java

```diff
@@ -68,7 +68,23 @@ public void testFailureLivenessCheck() throws Exception {
 
         JsonObject jsonResponse = getJSONPayload(conReady);
         JsonArray checks = (JsonArray) jsonResponse.get("checks");
-        assertEquals(1, checks.size());
+        assertEquals(2, checks.size());
+        assertEquals(true, checks.getJsonObject(0).getString("name").equals("failed-liveness-check") ||
+                           checks.getJsonObject(1).getString("name").equals("failed-liveness-check"));
+        assertEquals(jsonResponse.getString("status"), "DOWN");
+    }
+
+    @Test
+    public void testFailureCDIProducerLivenessCheck() throws Exception {
+        log("testCDIProducerLivenessCheck", "Testing the /health/live endpoint");
+        HttpURLConnection conReady = HttpUtils.getHttpConnectionWithAnyResponseCode(server1, LIVE_ENDPOINT);
+        assertEquals(FAILED_RESPONSE_CODE, conReady.getResponseCode());
+
+        JsonObject jsonResponse = getJSONPayload(conReady);
+        JsonArray checks = (JsonArray) jsonResponse.get("checks");
+        assertEquals(2, checks.size());
+        assertEquals(true, checks.getJsonObject(0).getString("name").equals("failed-cdi-producer-liveness-check") ||
```

Comment:
Since, you are checking the assetEquals for each name for each test case, it would be better and cleaner to add a helper method (e.g. checkIfHealthCheckNameExists) that cycles through the JsonArray object, and checks if the name e equals to the passed in name/s, in a for loop? If it does not exist, it will return false, where you just have to pass the boolean to assertTrue().